### PR TITLE
Fixing guides for gloo fed

### PR DESCRIPTION
**Pull request by ianmacclancy**
Updating walkthrough details where they are incorrect
related to : https://github.com/solo-io/solo-projects/issues/4327

### CODE REVIEW

File: docs/content/guides/gloo_federation/multicluster_rbac.md [hunk 1]
```diff
@@ -45,14 +45,14 @@ helm repo add gloo-fed https://storage.googleapis.com/gloo-fed-helm
 helm repo update
 
 # Upgrade your Gloo Edge Federation deployment
-# Assumes your install is called gloo-fed in the gloo-system namspace
-helm upgrade -n gloo-system gloo-fed gloo-fed/gloo-fed --set enableMultiClusterRbac=true
+# Assumes your install is called gloo in the gloo-system namspace on the cluster you have installed gloo-fed
+helm upgrade -n gloo-system gloo gloo-fed/gloo-fed --set enableMultiClusterRbac=true
```
Comment (jenshu) [inline docs/content/guides/gloo_federation/multicluster_rbac.md:49]: why this change, it should be ok for the release to be called `gloo-fed`(?)

File: projects/gloo/cli/pkg/cmd/demo/federation.go [hunk 2]
```diff
@@ -84,7 +84,7 @@ kubectl config use-context kind-"$1"
 
 # Install gloo-fed to cluster $1
 if [ "$5" == "" ]; then
-  glooctl install gateway enterprise --license-key=$4 --version=$3
+  glooctl install gateway enterprise --license-key=$4 --version=$3
```
Comment (jenshu) [inline projects/gloo/cli/pkg/cmd/demo/federation.go:87]: why was a whitespace added?

File: docs/content/guides/gloo_federation/multicluster_rbac.md (hunk 1 above)
Comment (ianmacclancy) [reply docs/content/guides/gloo_federation/multicluster_rbac.md:49]: This is just adding context, you spin up a few clusters as part of the setup. It's letting the user know the command on the next line makes an assumption about the name that may not be true to their setup. I am aligning the script with what the demo spins up.

File: projects/gloo/cli/pkg/cmd/demo/federation.go (hunk 2 above)
Comment (ianmacclancy) [reply projects/gloo/cli/pkg/cmd/demo/federation.go:87]: Accidental spacebar hit.

File: projects/gloo/cli/pkg/cmd/demo/federation.go [hunk 3]
```diff
@@ -114,7 +114,7 @@ kubectl patch settings -n gloo-system default --type=merge -p '{"spec":{"watchNa
 
 # Generate downstream cert and key
 openssl req -x509 -nodes -days 365 -newkey rsa:2048 \
-   -keyout tls.key -out tls.crt -subj "/CN=solo.io"
+   -keyout tls.key -out tls.crt -subj "/CN=solo.io"
```
Comment (jenshu) [inline projects/gloo/cli/pkg/cmd/demo/federation.go:117]: can you remove this one too?

**Review by jenshu — APPROVED** [review]
